src/Balloon.scala

```diff
@@ -0,0 +1,13 @@
+
+class Balloon{
+//minimal definition of a balloon. Method call sings 99 red balloons by nena
+    var howMany = 0
+    var color = ""
+    var inflate = true
+    def singIt(): Unit = {
+      var inflated = ""
+      if (inflate) inflated = "Floating in the summer sky" else inflated = "Go by"
```

Comment:
Update this using a Functional approach using pattern matching instead of IF-ELSE logic. FP starts with the data structure, inflate. Something like this:

inflate match {
      case true => println("Floating in the summer sky")
      case false => println("Go by")
    }